src/backends/cgpm_backend.py

```diff
@@ -17,6 +17,7 @@
 import itertools
 import json
 import math
+import operator, itertools
```

Comment:
One import per line, sorted alphabetically.

Reply:
Why?

Reply:
oh well, itertools is already there anyhow, and it's in the right place.

but I'm still curious about the source, importance, and transparency of all these rules.

Reply:
For source, most rules are from PEP 8---an "official" Python style guide---with certain spefifications from https://github.com/probcomp/bayeslite/blob/master/CONTRIBUTING.md.  The importance is having a consistent style guide to harmonize the contributions coming from various developers, which is common in software projects...